module/Controller.h

```diff
@@ -27,10 +27,21 @@ class Controller {
   void setLeftMotorPwm(const int pwm);
 
   /**
-   * 停止する
+   * モータを停止する
    */
   void stopMotor();
```

Comment:
アームのモータもあるしこの名前はよくなかったねゴメン
ちょっと面倒かもしれないけど`stopArmMoto()`と同じように修正お願いします
```suggestion
   * タイヤのモータを停止する
   */
  void stopWheelMotor();
```